examples/hitl/rearrange_v2/rearrange_v2.py

```diff
@@ -486,9 +589,12 @@ def sim_update(self, dt: float, post_sim_update_dict):
 
         #  Collect data.
         self._elapsed_time += dt
+        # TODO: Always record with non-human agent.
```

Comment:
What does this mean?

Reply:
In single-user and multi-user modes, we can skip recording of frames when there's no user input.

With the LLM agent, we'll most likely have to record every frame.